core/src/test/java/org/opensearch/sql/expression/operator/arthmetic/MathematicalFunctionTest.java

```diff
@@ -1804,6 +1804,72 @@ public void sign_missing_value() {
     assertTrue(sign.valueOf(valueEnv()).isMissing());
   }
 
+  /**
+   * Test sinh with byte value.
+   */
+  @ParameterizedTest(name = "sinh({0})")
+  @ValueSource(bytes = {-1, 1, 2})
+  public void sinh_byte_value(Byte value) {
+    FunctionExpression sinh = DSL.sinh(DSL.literal(value));
+    assertThat(sinh.valueOf(valueEnv()), allOf(hasType(DOUBLE), hasValue(Math.sinh(value))));
+    assertEquals(String.format("sinh(%s)", value), sinh.toString());
+  }
+
+  /**
+   * Test sinh with short value.
+   */
+  @ParameterizedTest(name = "sinh({0})")
+  @ValueSource(shorts = {-1, 1, 2})
+  public void sinh_short_value(Short value) {
+    FunctionExpression sinh = DSL.sinh(DSL.literal(value));
+    assertThat(sinh.valueOf(valueEnv()), allOf(hasType(DOUBLE), hasValue(Math.sinh(value))));
+    assertEquals(String.format("sinh(%s)", value), sinh.toString());
+  }
+
+  /**
+   * Test sinh with integer value.
+   */
+  @ParameterizedTest(name = "sinh({0})")
+  @ValueSource(ints = {-1, 1, 2})
```

Comment:
You might consider using numbers larger than bytes instead of these small numbers